### ISSUE TITLE
Create Bioconda Package for phantompeakqualtools

### ISSUE THREAD

**Issue by apeltzer** [nf-core/chipseq#14]

**Comment by apeltzer** [comment]
Related to https://github.com/nf-core/chipseq/issues/2

**Comment by apeltzer** [comment]
Isn't this one working fine too? 

https://github.com/hms-dbmi/spp

or do we need this (modified version?):

https://github.com/kundajelab/phantompeakqualtools

The "run_SPP.R" script could also go to the `bin` folder and we could simply use the CRAN version of the package then? 

@ewels @tiagochst What do you think? 

Just wanted to clarify this before starting packaging work!

**Comment by ewels** [comment]
Ok cool, this is coming together...

* SPP is already on bioconda: https://bioconda.github.io/recipes/r-spp/README.html
  * This is the same version that's [already packaged in bioconda](https://github.com/bioconda/bioconda-recipes/blob/b09381c6923dfbf25da845a56a6a227f30957f4b/recipes/r-spp/meta.yaml#L6)
* Phantompeakqualtools is the second link you posted. This is totally stand-alone to SPP.

It's the latter that I thought that we would have to package on bioconda. Whilst it would arguably still be a nice thing to do for the community, I hadn't realised that it really is just a single R script. As you say, it would be trivial to just drop this into the pipeline `bin` directory. Maybe we should just stick with this.

The only thing that makes me nervous is the bit you mention on the [phantompeakqualtools readme](https://github.com/kundajelab/phantompeakqualtools#files):

> `spp_1.14.tar.gz` : **modified** SPP peak-caller package (The original SPP-peak caller package was written by Peter Kharchenko [2], https://github.com/hms-dbmi/spp)

The term _modified_ worried me a bit. But I just downloaded it and it has the same md5 as the [v1.14 release](https://github.com/hms-dbmi/spp/releases/tag/1.14) of SPP on your first link. So I don't think it actually is modified in the phantompeakqualtools repo.

### In summary

Let's just stick `run_SPP.R` into the pipeline `bin` directory and describe what we did on the readme. We should probably pin the bioconda SPP version to 1.14 so that it ties in with the phantompeakqualtools release.

Now or another time we could package this script onto bioconda, but I don't think that it really adds anything for us here.

**Comment by apeltzer** [comment]
Exactly what worried me a bit too! But I'd also say we just place the `run_SPP.R` in `bin` of the pipeline and use the bioconda package that is already available and then just test if things work? That would be probably the cleanest thing? 

Regarding the genomes for ngsplot, I'll work on this asap in #15.

**Comment by apeltzer** [comment]
This has been done on the `bioconda` branch. Closing this now :-)
